cypress/integration/profile/edit-profile-email.js

```diff
@@ -0,0 +1,41 @@
+const { beforeEach } = require('mocha');
+
+describe ('Edit Profile Email', function() {
+  before('Login', function() {
+    cy.Login()
+  })
+           
+  it('Edit Email',function() {
+    cy.get(':nth-child(4) > .nav-link').click(); 
+    cy.contains("Edit Profile Settings").click()
+    cy.get('[formcontrolname=email]').clear().type("santiago.lopezqa+3@wolox.com.ar"); 
+    cy.get('[formcontrolname=password]').clear().type("Wolox1189!"); 
```

Comment:
Please move this credentials to environment variable.
https://docs.cypress.io/guides/guides/environment-variables.html